### PR TITLE
bring back the moon

### PR DESCRIPTION
**Pull request by othane**
Earth misses its moon, lets bring it back

### CODE REVIEW

File: pyg_planets1.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def game_loop():
       qY = int(Oy - pY)
       pygame.draw.circle(screen, PLANET_COLR[p], 
                           [qX, qY], int(PLANET_SIZE[p]), 0)     #draw planet
-      if p == 2:  # earth
+      if p == 'Earth':  # earth
```
Comment (othane) [inline pyg_planets1.py:115]: I think this is the change you were talking about

File: pyg_planets1.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def game_loop():
         pl_trail_X[p].append(qX)
         pl_trail_Y[p].append(qY)
         #print("%%%", ctr, len(pl_trail_X[p]))
-      if p == 2 and ctr%5 == 0:   #moon trail   
+      if p == 'Earth' and ctr%5 == 0:   #moon trail
```
Comment (othane) [inline pyg_planets1.py:124]: but I think this one also needs to change ?

File: pyg_planets1.py (hunk 1 above)
Comment (kdt777) [reply pyg_planets1.py:115]: also moon trail (line 124)

**Comment by othane** [comment]
Changes already done in main branch

**Pull request closed without merging**